README.md

```diff
@@ -21,6 +21,7 @@ To develop on this project, you should have the following installed:
 * Python 3.8.* (We recommend using [pyenv](https://github.com/pyenv/pyenv))
```

Comment:
We'll also want to note down about uploading the docker images/creating ECR repositories...

Reply:
@ciaranevans I'm going to skip over this for now since this will be superseded by Ryan's suggested switch to Conda and dynamic deps for environment builds.